caravel/assets/visualizations/filter_box.jsx

```diff
@@ -10,22 +10,80 @@ import '../stylesheets/react-select/select.less';
 
 import './filter_box.css';
 
+const TIME_CHOICES = [
+  '1 hour ago',
+  '12 hours ago',
+  '1 day ago',
+  '7 days ago',
+  '28 days ago',
+  '90 days ago',
+  '1 year ago',
+];
+
+const propTypes = {
```

Comment:
i like how you moved these to the top of the file. it's a pattern i use too & find super useful when reading a component file. 👌